docs/contributing.md

```diff
@@ -20,17 +20,17 @@ Please submit bug requests and feature requests through Github as issues. No spe
 
 Please see the information provided in [SECURITY.md](SECURITY.md)
 
-## Development Process for a New Score or Metrics
+## Development Process for a New Score or Metric
 
 A new score or metric should be developed on a separate feature branch, rebased against the main branch. Each merge request should include:
 
  - The implementation of the new metric or score in xarray, ideally with support for pandas and dask
  - 100% unit test coverage
  - A tutorial notebook showcasing the use of that metric or score, ideally based on the standard sample data
- - API documentation (docstrings) using [Napoleon (google)](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style, making sure to clearly explain the use of the metrics
- - A reference to the paper which described the metrics, added to the API documentation
+ - API documentation (docstrings) using [Napoleon (google)](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) style, making sure to clearly explain the use of the metric
+ - A reference to the peer-reviewed paper which describes the metric, added to the API documentation
```

Comment:
Some scores do not have an original peer-reviewed paper (e.g. mean squared error) and a reasonable reference must suffice. Where there is an original or definitive paper, it should be cited. We should make sure the reference matches what we actually built (and vice versa). I'm not sure exactly the best way to express all that here. @BethEbert

Reply:
How about "a reference to a peer-reviewed paper or other reasonable reference (e.g. a textbook)..."

Reply:
Another thought. In cases when there is no specific or authoritative reference, and another reference is being used instead, should contributors be encouraged to cite an open source? Not everyone has institutional access to paywalled articles and books. (For instance, I don't). In many cases, there are good, peer-reviewed survey/review/overview articles that are open access. (I understand people will need to continue citing paywalled references when they are the authoritative reference).